compiler/main/driver.cpp

```diff
@@ -1528,6 +1529,7 @@ static ArgumentDescription arg_desc[] = {
  {"dyno-gen-lib", ' ', "<path>", "Specify files named on the command line should be saved into a .dyno library", "P", NULL, NULL, addDynoGenLib},
  {"dyno-gen-std", ' ', NULL, "Generate a .dyno library file for the standard library", "F", &fDynoGenStdLib, NULL, setDynoGenStdLib},
  {"dyno-verify-serialization", ' ', NULL, "Enable [disable] verification of serialization", "N", &fDynoVerifySerialization, NULL, NULL},
+ {"dyno-no-break-error", ' ', NULL, "Disable breakpoint for user errors from the frontend", "N", &fDynoNoBreakError, NULL, NULL},
```

Comment:
I don't know how important it is practically, but I think this would match our conventions better as `--[no-]dyno-break-error`